common/sshtools.py

```diff
@@ -42,7 +42,7 @@ class SSH(mount.MountControl):
 
     CHECK_FUSE_GROUP = True
 
-    def __init__(self, cfg = None, profile_id = None, hash_id = None, tmp_mount = False, parent = None, symlink = True, **kwargs):
+    def __init__(self, cfg = None, profile_id = None, hash_id = None, tmp_mount = False, parent = None, symlink = True, read_only = True, **kwargs):
```

Comment:
`read_only` need to be written into `unmount` info. This is would be done by `self.setattr_kwargs('read_only', False, **kwargs)` (in line 67). This is some 'magic' (actually no magic at all just crappy, undocumented code from my lazy self :wink:) that will set `self.read_only` and also put it into `unmount` info. 

`unmount` info is responsible for sharing the mount-point. 'Magic' behind this is that BIT uses a `crc32` hash of that file as mount-point name. If there is a second process trying to mount with the exact same args it will find this mount-point already mounted and just add its locks and symlink.

So don't put `read_only` into `__init__` args. `**kwargs` will fetch this. And also no `self.read_only = read_only`. Just add the `self.setattr_kwargs(...)` line in 67.